src/Events/Repository/CloudEventRepository.cs

```diff
@@ -49,6 +51,21 @@ public async Task CreateAppEvent(CloudEvent cloudEvent)
             pgcom.Parameters.Add(new NpgsqlParameter("cloudevent", cloudEvent.Serialize()) { Direction = System.Data.ParameterDirection.Input });
 
             await pgcom.ExecuteNonQueryAsync();
+
+            await using NpgsqlCommand pgcom2 = new NpgsqlCommand(string.Format(insertEventSql, serializedCloudEvent), conn);
```

Comment:
We should avoid building a SQL statement with any form of string manipulation. Always use parameters. 

When it comes to parameters we've usually named them, but @vgr has found that using ordering is best. Not sure how much it matters though.

Reply:
I find named parameters to be more readable and safer when refactoring. In this case, I would expect the performance difference should be non-measurable.

Reply:
Given that it's only a single parameter and we have the full sql statement available I'm going with the numbered parameter here, the property name should be telling enough w.r.t readability / refactoring